test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DisasterRecovery/BackupServiceTests.cs

```diff
@@ -53,7 +53,7 @@ public class BackupServiceTests
                 OwnerUri = liveConnection.ConnectionInfo.OwnerUri
             };
 
-            await DisasterRecoveryService.HandleBackupConfigInfoRequest(dbParams, requestContext.Object);
+            await DisasterRecoveryService.Instance.HandleBackupConfigInfoRequest(dbParams, requestContext.Object);
```

Comment:
>DisasterRecoveryService.Instance [](start = 18, length = 32)

Even for integration tests, using a `new DisasterRecoveryService()` instead of the instance is preferred. As mentioned, if there is shared state in the future this can mess up tests that are independent of each other.